Cargo.toml

```diff
@@ -79,9 +79,9 @@ serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 slog = { version = "2.7", features = [
-  "dynamic-keys",
-  "max_level_trace",
-  "release_max_level_info",
+    "dynamic-keys",
+    "max_level_trace",
+    "release_max_level_info",
```

Comment:
No matter the editor, I am getting these diffs....

Reply:
Huh, it *might* be a `cargo fmt` change? There have been a few similar things in the past. 
FWIW, I tend to run the latest stable, but you'll notice that the version in `#RUST_VER` tends to lag behind a bit.